crates/voicevox_core_node_api/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name = "voicevox_core_node_api"
+version = "0.0.0"
```

このリポジトリでの他のpackageに合わせると、こうなると思います。

参考: #326

```suggestion
name = "voicevox_core_node_api"
version = "0.0.0"
edition.workspace = true
publish.workspace = true
```